src/Security/Authorization/Policy/src/PolicyServiceCollectionExtensions.cs

```diff
@@ -18,15 +18,25 @@ public static class PolicyServiceCollectionExtensions
         /// </summary>
         /// <param name="services">The <see cref="IServiceCollection" /> to add services to.</param>
         /// <returns>The <see cref="IServiceCollection"/> so that additional calls can be chained.</returns>
-        public static IServiceCollection AddAuthorizationPolicyEvaluator(this IServiceCollection services)
+        [Obsolete("Please use .AddAuthorizationPolicy() extension method instead")]
```

Comment:
I don't think we gain very much by renaming this now, AddAuthorization already does the right thing, which is the main method that people call, so lets just leave this with the admittedly less good specific name.

Reply:
ok, changed it back